src/popup/components/SelectAsset.vue

```diff
@@ -21,6 +21,7 @@ export default {
   },
   props: {
     value: { type: Object, default: null },
+    canSelectAsset: { type: Boolean, default: true },
```

Comment:
In case of the boolean props I suggest to not use the `true` as the default as it looks bad in the templates when we want to disable something. I suggest to inverse the logic. So maybe `disabled` prop would be better?